ansible/configs/ocp4-cluster/default_vars.yml

```diff
@@ -0,0 +1,195 @@
+###### VARIABLES YOU SHOULD CONFIGURE FOR YOUR DEPLOYEMNT
+###### OR PASS as "-e" args to ansible-playbook command
+
+# The name of the agnosticd config to deploy
+env_type: ocp4-cluster
+
+# The output_dir holds all of the files generated during the deployment
+# This includes generated Heat templates, SSH config, SSH keys
+# This must be an absolute path and no vars (like $HOME or ~)
+output_dir: /tmp/output_dir
+
+# The {{ guid }} is used everywhere and it is what differentiates otherwise
+# identical environments. Make this unique. Usually they are 4 characters, but
+# it can be any reasonable length.
+guid: notset
+
+# The name of the OpenShift cluster that will be deployed.
+# This is primarily used if you want to automate the OpenShift deployment.
+cluster_name: cluster-{{ guid }}
+
+# Used to add metadata (tags) to OpenStack objects created
+project_tag: "{{ env_type }}-{{ guid }}"
+
+# Why is this config being deployed? Override for specific environments
+# Some valid: development, ilt, production, event
+purpose: development
+
+# Enable this if you want to create a user on the bastion
+# Mutually exclusive with {{ install_ipa_client }}
+install_student_user: true
+
+# This should be overwritten based on the user ordering the catalog item
+# It will be used by the bastion-student-user role and created on the bastion
+student_name: lab-user
+
+# Enable this if you want to use IPA for user authentication.
+# Mutually exclusive with {{ install_student_user }}
+install_ipa_client: false
+
+# TODO: What does this really do besides run the role?
+set_env_authorized_key: true
+env_authorized_key: "{{guid}}key"
+key_name: "default_key_name"
```

Comment:
key_name should not be needed anymore when `cloud_provider == osp` thanks to this recent commit: https://github.com/redhat-cop/agnosticd/commit/926a092587159ebcc816cbdbe39db71b1f5b702b#diff-beb8a8b8bdf73e75e38e21e91cb5ef28R2